src/quickgui/plugin/editor/qgsquicktextedit.qml

```diff
@@ -72,6 +72,8 @@ Item {
     visible: height !== 0
     anchors.left: parent.left
     anchors.right: parent.right
+    font.pointSize: 14 * QgsQuick.Utils.dp
```

Comment:
not sure if font size should be multiplied by dp, last time I tried and it ended up in unreadable thing ... have you tried it on tablet? was it readable?

Reply:
Just copied from TextField above this Multiline TextField to have same font size. However, agree that it its not necessary to have QgsQuick.Utils.dp there, but its not messing up form.